src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4062,7 +4062,7 @@ You should consider suppressing the warning only if you're sure that you don't w
     <value>Named argument '{0}' is used out-of-position but is followed by an unnamed argument</value>
   </data>
   <data name="ERR_DefaultValueUsedWithAttributes" xml:space="preserve">
-    <value>Cannot specify default parameter value in conjunction with DefaultParameterAttribute or OptionalAttribute</value>
+    <value>Cannot specify default parameter value in conjunction with DefaultParameterValueAttribute or OptionalAttribute</value>
```

Comment:
>DefaultParameterValueAttribute [](http://example.com/codeflow?start=70&length=30)

Is this change related to the goal of the PR?

Reply:
It looks like this change is unrelated. I think it should be removed from this PR.